guides/source/active_record_callbacks.md

```diff
@@ -647,7 +981,10 @@ class PictureFile < ApplicationRecord
 end
 ```
 
-WARNING. Using both `after_create_commit` and `after_update_commit` with the same method name will only allow the last callback defined to take effect, as they both internally alias to `after_commit` which overrides previously defined callbacks with the same method name.
+WARNING. Using both `after_create_commit` and `after_update_commit` with the
+same method name will only allow the last callback defined to take effect, as
+they both internally alias to `after_commit` which overrides previously defined
+callbacks with the same method name.
 
 ```ruby
 class User < ApplicationRecord
```

Comment:
Let's update this example to remove the `@` instance var in favor of just a local var too.

Reply:
✅